modules/api/src/main/java/org/apache/ignite/tx/Transaction.java

```diff
@@ -46,4 +47,20 @@
      * @return The future.
      */
     CompletableFuture<Void> rollbackAsync();
+    
+    /**
+     * Enlists SQL session into a transaction.
+     *
+     * @param session SQL session.
+     * @return Enlisted session.
+     */
+    Session wrap(Session session);
```

Comment:
"Enlists SQL session into a transaction."

So, may be method should have name `enlist`?